xian_test.go

```diff
@@ -0,0 +1,58 @@
+package xian
+
+import (
+	"testing"
+)
+
+func TestBiunigrams(t *testing.T) {
+	result := Biunigrams("abc dあいbCh")
+	if len(result) != 13 {
+		t.Errorf("len(result) exected:%d, but was:%d\n", 13, len(result))
+	}
+
+	expectedGrams := []string{
+		"ab", "bc", "dあ", "あい", "いb", "ch",
+		"a", "b", "c", "d", "あ", "い", "h"}
+	for _, gram := range result {
```

Comment:
Did you write this for-loop because of random order of result?
please sort result, so that you can assert by fixed order.